content/en/cosign/signing_with_blobs.md

```diff
@@ -95,3 +48,51 @@ tlog entry created with index: 965333
 Signature wrote in the file sig
 Certificate wrote in the file cert.pem
 ```
+
+
+## Signing with a key
+
+You may specify your own keys for signing.  You will need the password for the private key to sign:
+
+```shell
+$ cosign sign-blob --key cosign.key README.md
+Using payload from: README.md
+Enter password for private key:
+MEQCIAU4wPBpl/U5Vtdx/eJFgR0nICiiNCgyWPWarupH0onwAiAv5ycIKgztxHNVG7bzUjqHuvK2gsc4MWxwDgtDh0JINw==
+```
+
+This supports all the same flags and features as `cosign sign`, including KMS support, hardware tokens, and keyless signatures.
```

Comment:
Should we link to the "signing with self managed keys" section here instead of duplicate the command and output? I'm fine with the duplication for what it's worth, I think we should link to the other section regardless.